src/store/initial-state.js

```diff
@@ -32,25 +32,25 @@ export const createInitialState = () => ({
   },
   visible: {
     code: false,
-    exportBtn: true,
     exportModal: false,
     graph: true,
-    labelBtn: true,
-    layerBtn: true,
     metadataModal: false,
     miniMap: true,
-    miniMapBtn: true,
     modularPipelineFocusMode: null,
```

Comment:
[Nit] Can be ignored, would like to see this as false/true in future instead of null. Or if there is any reason for this to be null, please let me know. Thank you